salt/modules/cmdmod.py

```diff
@@ -951,6 +971,13 @@ def run(cmd,
         Be absolutely certain that you have sanitized your input prior to using
         python_shell=True
 
+    :param list success_exit_codes: This parameter will be allow a list of
+        non-zero return codes that should be considered a success.  If the
+        return code  returned from the run matches any in the provided list,
+        the return code will be overridden with zero.
+
+      .. versionadded:: Fluorine
+
```

Comment:
NOTE: The above two comments should be addressed in all places in the docstrings where they were copied.